doc/development/rfc/release_policy.md

```diff
@@ -0,0 +1,118 @@
+as soon as possible with software stability and compatibility and without
+a need to wait for a major release.
+
+Both major and minor releases may include bugfixes which may or may not be
+already included in a micro release of the previous major or minor release.
+
+A special effort is put into keeping backwards compatibility.
+While this is a requirement for minor version, the community makes extra effort
+even for major release to maximize backwards compatibility so that users can
+conveniently transition to new versions.
+
+### Micro
+
+Micro releases should contain only bugfixes.
+Their purpose is to fix issues with a particular major or minor release.
+Providing both stability and fixes to the users.
+
+A minor release should be considered instead of a micro release when
+a bugfix is large, a bugfix has unclear side-effects, or it is not clear
+if it is bugfix or a feature given that, in practice, the line between
+a bugfix and a feature is not completely clear.
+
+### Pre-releases and Daily Builds
+
+There can be zero or more release candidates, i.e., pre-releases,
+for each major, minor, or micro release. The specifics are determined
+by the Release Procedure RFC.
+
+There can be daily or per-commit frequent builds of
+which can serve as preview versions or version for testing.
+Other frequencies are permissable but not recommended.
+
+### Post-releases and Build Versions
+
+There can be subsequent builds of a specific release
+if build or packaging process failed and needs fixing.
+However, there shouldn't be any post-releases, i.e.,
+if there is a change in the code base to fix a flaw in a release,
+a new micro release should be created.
+
+## Frequency
+
+One major or minor release should be released at least once a year.
+More then one major or minor release may be released
+if the contributors consider that necessary or advantageous.
+There is no requirement for frequency of major releases, but
+the practice and rhythm so far was to release major version
+before the minor version number reaches value ten which translates
+to a major release every five to ten years.
+
+There can be zero or more micro releases per year for different major or minor
+release based on the needs of the community practically driven by the number
+of fixes accumulated for a particular release.
+
+## Maintenance
+
+We have one actively updated, maintained, and supported release series
+which is started by a release of a new major or minor version.
+We release micro versions in this series throughout the year as needed.
+
+Once a new major or minor version is released, the old release series goes to
+a low maintenance mode and no further releases are planned.
+However, any low maintenance series can be updated on demand.
+If you submit a patch to fix a bug, we will likely accept it and
+create a new release when there is enough changes accumulated
+or support you in creating the release as a temporary release manager.
```

Comment:
```suggestion
If a contributor submits a patch to fix a bug, it will be likely accepted and
a new release will be created when there are enough changes accumulated.
Alternatively the contributor might receive support in creating the release as
a temporary release manager.
```

Reply:
I changed to third person to keep the tone consistent